src/metaswitch/clearwater/config_manager/config_access.py

```diff
@@ -792,8 +807,17 @@ def print_diff_and_syslog(config_type, config_1, config_2):
         additions.remove(item)
 
     if additions or deletions or moved:
-        header = "Configuration file change: user {} has modified {}.".format(
-            get_user_name(),
+
+        # If this is being run via ssh then we won't be able to get the user
+        # name. We'll already have logged elsewhere, so just log the changes.
```

Comment:
Could you please say where exactly? When I first read this I thought that we already logged it somewhere in this function/file so it would be good to have the context of where we log to remind our future selves :-)